README.md

```diff
@@ -151,6 +151,12 @@ Options for the [less](http://lesscss.org) compiler (`tree.toCss`).
 
 **Note:** The options `sourceMap` and `cleancss` are not supported. An exception will be thrown when used.
 
+###### cssVars
```

Comment:
As the option is not relevant for the less compiler, I would rather have it next to the other custom options such as "rtl" or "rootPaths".

Reply:
Ok